tests/aequilibrae/paths/test_network_skimming.py

```diff
@@ -1,40 +1,75 @@
 from unittest import TestCase
 import numpy as np
-from aequilibrae.paths import Graph
 from aequilibrae.paths.network_skimming import NetworkSkimming
 from aequilibrae.paths import skimming_single_origin
 from aequilibrae.paths.results import SkimResults
 from aequilibrae.paths.multi_threaded_skimming import MultiThreadedNetworkSkimming
-from aequilibrae.paths import binary_version
+import os
+from shutil import copytree, rmtree
+import uuid
+from tempfile import gettempdir
+from aequilibrae.project import Project
+
+from ...data import siouxfalls_project
 
 # Adds the folder with the data to the path and collects the paths to the files
 from ...data import test_graph
 
 
 class TestNetwork_skimming(TestCase):
+    def setUp(self) -> None:
+        os.environ['PATH'] = os.path.join(gettempdir(), 'temp_data') + ';' + os.environ['PATH']
+
+        self.proj_dir = os.path.join(gettempdir(), uuid.uuid4().hex)
+        copytree(siouxfalls_project, self.proj_dir)
+
+        self.project = Project()
+        self.project.open(self.proj_dir)
+        self.network = self.project.network
+        self.curr = self.project.conn.cursor()
+
+    def tearDown(self) -> None:
+        self.project.close()
+        del self.curr
+        try:
+            rmtree(self.proj_dir)
+        except Exception as e:
+            print(f'Failed to remove at {e.args}')
+
     def test_network_skimming(self):
-        # graph
-        g = Graph()
-        g.load_from_disk(test_graph)
-        g.__version__ = binary_version
-        g.set_graph(cost_field="distance")
-        g.set_skimming("distance")
+
+        self.network.build_graphs()
+        graph = self.network.graphs['c']
+        graph.set_graph(cost_field="distance")
+        graph.set_skimming("distance")
 
         # skimming results
         res = SkimResults()
-        res.prepare(g)
-
+        res.prepare(graph)
         aux_res = MultiThreadedNetworkSkimming()
-        aux_res.prepare(g, res)
-        _ = skimming_single_origin(26, g, res, aux_res, 0)
+        aux_res.prepare(graph, res)
+        _ = skimming_single_origin(12, graph, res, aux_res, 0)
 
-        skm = NetworkSkimming(g, res)
+        skm = NetworkSkimming(graph)
         skm.execute()
 
-        tot = np.nanmax(res.skims.distance[:, :])
-
+        tot = np.nanmax(skm.results.skims.distance[:, :])
         if tot > 10e10:
```

Comment:
not sure the standard but I would have a constant defined with this number

Reply:
This is largely arbitrary, as paths not found return something closer to infinite.  I have changed to something a little more sensible